source-hubspot-native/source_hubspot_native/api.py

```diff
@@ -271,14 +460,260 @@ async def fetch_recent_tickets(
     log: Logger, http: HTTPSession, since: datetime, cursor: PageCursor
 ) -> tuple[Iterable[tuple[datetime, str]], PageCursor]:
 
-    url = f"{HUB}/crm-objects/v1/change-log/tickets"
-    params = {"timestamp": int(since.timestamp() * 1000) - 1}
```

Comment:
Is there not a way to pass the timestamp parameter anymore? Do we need to filter the result on `since` as we will now I assume get every single ticket each time?

Reply:
So, when developing this i had 2 main thoughts in my head ( Hubspot [docs](https://legacydocs.hubspot.com/docs/methods/tickets/get-ticket-changes) for reference):

1. If im using this endpoint to trigger which tickets have changed, i will not get all tickets. They do not specify what is a change, and i dont think a  ticket creation would reach this.
2. Even if it did, when i select the timestamp i would never allow for a """full_refresh""" pattern where i get all data. From what i understood, the since is a way of us understading where the last data parsed was set in history, not as means of getting a specific timeframe

Reply:
> 2\. the `since` is a way of us understading where the last data parsed was set in history, not as means of getting a specific timeframe

just further explaining this quote, i believe the `timestamp` parameter would return to me 'timestamp - 24 >= data => timestamp`  as is cited in the docs